### PR TITLE
Filter function: blur()

### PR DESCRIPTION
**Pull request by estelle**
Add SVG to blur function

(I will be adding the SVG to all filter functions. This is the first)

### CODE REVIEW

**Review by teoli2003 — CHANGES_REQUESTED** [review]
I agree that mentioning the SVG equivalent and adding it to the example for comparison is helpful.

I have trouble with a whole section about the SVG equivalent: it makes the page more about it than about its theme, the CSS `blur()` function. This section should be morphed into a simple mention with a link to the SVG page. The page is about the CSS `blur()` function and not a guide about how to blur an image.

**Comment by estelle** [comment]
> I agree that mentioning the SVG equivalent and adding it to the example for comparison is helpful.
> 
> I have trouble with a whole section about the SVG equivalent: it makes the page more about it than about its theme, the CSS `blur()` function. This section should be morphed into a simple mention with a link to the SVG page. The page is about the CSS `blur()` function and not a guide about how to blur an image.

I moved the content over from the `filter` property page, where it made less sense to me. It is copied from the spec and is  a way of understanding how the underlying blur function works. Also, there is no SVG page explaining blur. This moving of the SVG edits are already a rabbit hole; I don't want to start creating SVG pages also, because then i would have to fix all the SVG pages that say that attributes( like  "slope") are deprecated when they're not.... another rabbit hole.

**Comment by dipikabh** [comment]
I agree with @teoli2003's argument for not adding the "SVG filter" section here.

The "SVG filter" section seems to be specific to the `<feGaussianBlur>` filter element. How about moving this section to the [feGaussianBlur](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feGaussianBlur) page? A link on the `blur()` function page can point to this section on the feGaussianBlur page. That would also help to link back from feGaussianBlur page to blur() page.

You can probably retain the SVG filter example here.

"Setting a blur with pixels and with rem" would need to be moved back to the "Examples" section.

File: files/en-us/web/css/filter-function/blur/index.md [hunk 1]
```diff
@@ -13,16 +13,16 @@ The **`blur()`** [CSS](/en-US/docs/Web/CSS) [function](/en-US/docs/Web/CSS/CSS_F
...
 
 ### Parameters
 
 - `radius`
-  - : The radius of the blur, specified as a {{cssxref("&lt;length&gt;")}}. It defines the value of the standard deviation to the Gaussian function, i.e., how many pixels on the screen blend into each other; thus, a larger value will create more blur. A value of `0` leaves the input unchanged. The initial value for {{Glossary("interpolation")}} is `0`.
-
-## Examples
+  - : The radius of the blur, specified as a {{cssxref("&lt;length&gt;")}}. It defines the value of the standard deviation to the Gaussian function, i.e., how many pixels on the screen blend into each other; thus, a larger value will create more blur. A value of `0` leaves the input unchanged. The initial value for {{Glossary("interpolation")}} is `0`. Percentage values are invalid.
 
 ### Setting a blur with pixels and with rem
```
Comment (dipikabh) [inline files/en-us/web/css/filter-function/blur/index.md:27]:
```suggestion
### Applying blur by using units
```

IMO this should stay in "Examples"

File: files/en-us/web/css/filter-function/blur/index.md [hunk 2]
```diff
@@ -32,6 +32,83 @@ blur(8px)      /* Blur with 8px radius */
...
+
+```css
+filter: blur(1.1px);
+filter: url(#blur11); /* with embedded SVG */
+filter: url(folder/fileName.svg#blurr11); /* external svg filter definition */
+```
+
+## Examples
+
+This example shows three images: the image with a `blur()` filter function applied, the image with the equivalent SVG blur function applied, and the original images for comparison:
```
Comment (dipikabh) [inline files/en-us/web/css/filter-function/blur/index.md:57]: This example can be given a heading like "Applying blur by using an SVG filter"

**Review by dipikabh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**